tests/e2e/utils/kubeflow_installation.py

```diff
@@ -284,28 +284,34 @@ def configure_kubeflow_pipelines(
     if credentials_option == "static":
         return
 
-    cfg = load_yaml_file(file_path="./utils/pipelines/config.yaml")
-    IAM_ROLE_ARN_FOR_IRSA = cfg["pipeline_oidc_role"]
+    cfg = load_yaml_file(file_path="./utils/rds-s3/metadata.yaml")
+    IAM_ROLE_ARN_FOR_IRSA = cfg["S3"]["roleArn"]
 
     if installation_option == "kustomize":
         CHART_EXPORT_PATH = "../../awsconfigs/apps/pipeline/s3/service-account.yaml"
         USER_NAMESPACE_PATH = (
             "../../awsconfigs/common/user-namespace/overlay/profile.yaml"
         )
+        exec_shell(
+            f'yq e \'.metadata.annotations."eks.amazonaws.com/role-arn"="{IAM_ROLE_ARN_FOR_IRSA}"\' '
+            + f"-i {CHART_EXPORT_PATH}"
+        )
+        exec_shell(
+            f'yq e \'.spec.plugins[0].spec."awsIamRole"="{IAM_ROLE_ARN_FOR_IRSA}"\' '
+            + f"-i {USER_NAMESPACE_PATH}"
+        )
 
     else:
-        CHART_EXPORT_PATH = f"{installation_paths}/templates/ServiceAccount/ml-pipeline-kubeflow-ServiceAccount.yaml"
-        USER_NAMESPACE_PATH = "../../charts/common/user-namespace/templates/Profile/kubeflow-user-example-com-Profile.yaml"
-
-    exec_shell(
-        f'yq e \'.metadata.annotations."eks.amazonaws.com/role-arn"="{IAM_ROLE_ARN_FOR_IRSA}"\' '
-        + f"-i {CHART_EXPORT_PATH}"
-    )
-    exec_shell(
-        f'yq e \'.spec.plugins[0].spec."awsIamRole"="{IAM_ROLE_ARN_FOR_IRSA}"\' '
-        + f"-i {USER_NAMESPACE_PATH}"
-    )
-
+        CHART_EXPORT_PATH = f"{installation_paths}/values.yaml"
+        USER_NAMESPACE_PATH = "../../charts/common/user-namespace/values.yaml"
+        exec_shell(
+            f'yq e \'.irsa.roleArn="{IAM_ROLE_ARN_FOR_IRSA}"\' '
```

Comment:
this is a bit confusing we already have
```
s3:
  bucketName:
  minioServiceHost: s3.amazonaws.com
  minioServiceRegion:
```

roleArn should go under s3 instead of a separate irsa key
```
s3:
  bucketName:
  minioServiceHost: s3.amazonaws.com
  minioServiceRegion:
  roleArn:
```

Reply:
will resolve in separate pr